pkg/spec3/fuzz.go

```diff
@@ -169,82 +170,85 @@ var OpenAPIV3FuzzFuncs []interface{} = []interface{}{
 		c.Fuzz(&v.VendorExtensible)
 	},
 	func(v *spec.Extensions, c fuzz.Continue) {
-		*v = spec.Extensions{}
```

Comment:
I remember refactoring to this new version after a [comment](https://github.com/kubernetes/kube-openapi/pull/370#discussion_r1119160338) around the *v == nil case. Is this causing round tripping problems?

Reply:
No this one was actually just because it made me feel weird to set *v to a value to unset it later if unneeded. It felt more natural to set it when needed.